releases/release-1.18/README.md

```diff
@@ -0,0 +1,71 @@
+# Kubernetes 1.15
+
+
+#### Links
+
+* [This document](https://git.k8s.io/sig-release/releases/release-1.18/README.md)
+* [Release Team](https://git.k8s.io/sig-release/releases/release-1.18/release_team.md)
+* [Meeting Minutes]() (join [kubernetes-sig-release@] to receive meeting invites)
+* [v1.18 Release Calendar][k8s118-calendar]
+* Contact: [#sig-release] on slack, [kubernetes-release-team@] on e-mail
+* [Internal Contact Info]() (accessible only to members of [kubernetes-release-team@])
+
+#### Tracking docs
+
+* [Enhancements Tracking Sheet]()
+* [Bug Triage Tracking Sheet]()
+* [CI Signal Report]()
+* [Retrospective Document]()
+* [kubernetes/sig-release v1.18 milestone][]
+
+#### Guides
+
+* [Targeting Issues and PRs to This Milestone](https://git.k8s.io/community/contributors/devel/sig-release/release.md)
+* [Triaging and Escalating Test Failures](https://git.k8s.io/community/contributors/devel/sig-testing/testing.md#troubleshooting-a-failure)
+
+## tl;dr
+
+The 1.16 release cycle is proposed as follows:
+
+- **Monday, January 06**: Week 1 - Release cycle begins
+- **Tuesday, January 28**: Week 4 - [Enhancements Freeze]
+- **Thursday, March 05**: Week 9 - [Code Freeze]
+- **Monday, March 16**: Week 11 - Docs must be completed and reviewed
+- **Tuesday, March 24**: Week 12 - Kubernetes v1.18.0 released
+
+
+## Timeline
+
+
+| **What** | **Who** | **Jan** | **Feb** | **Mar** | **WEEK** | **CI SIGNAL** |
+| --- | --- | --- | --- | --- | --- | --- |
+| Start of Release Cycle | Lead | Mon 6 | | | week 1 | [master-blocking] |
+| Schedule finalized | Lead | Fri 10 | | | | |
+| Team finalized | Lead | Fri 17 | | | week 2 | |
+| Start Enhancements Tracking | Enhancements Lead | Tue 21 | | | week 3 | |
+| 1.18.0-alpha.1 released | Branch Manager | Tue 21 | | | | |
+| Start Release Notes Draft | Release Notes Lead | Tue 28 | | | week 4 | |
+| **Begin [Enhancements Freeze]** (EOD PST) | Enhancements Lead | Tue 4 | | | week 5 | [master-blocking], [master-informing] |
```

Comment:
```suggestion
| **Begin [Enhancements Freeze]** (EOD PST) | Enhancements Lead | Tue 28 | | | week 4 | [master-blocking], [master-informing] |
```
I think `Tue 4` is meant to be `Tue 28` for both this line and the next.  Unless this is week 5 in Feb and the TL;DR date above is wrong 😬

Reply:
Moving right a column into February as I believe this to be 2/4

```suggestion
| **Begin [Enhancements Freeze]** (EOD PST) | Enhancements Lead | | Tue 4 | | week 5 | [master-blocking], [master-informing] |
```